pom.xml

```diff
@@ -57,39 +57,98 @@
-                            org.apache.felix.inventory;resolution:=optional,
-                            *
-                        </Import-Package>
-                        <Export-Package>
-                        	org.apache.sling.event.jobs,
-                        	org.apache.sling.event.jobs.consumer,
-                        	org.apache.sling.event.jobs.jmx
-                        </Export-Package>
-                       <DynamicImport-Package>
-                            javax.servlet,
-                            javax.servlet.http,
-                            org.apache.felix.inventory
-                        </DynamicImport-Package>
-                        <Sling-Nodetypes>
-                            SLING-INF/nodetypes/event.cnd
-                        </Sling-Nodetypes>
-                        <Sling-Namespaces>
-                            slingevent=http://sling.apache.org/jcr/event/1.0
-                        </Sling-Namespaces>
-                        <Embed-Dependency>
-                            jackrabbit-jcr-commons;inline="org/apache/jackrabbit/util/ISO9075.*|org/apache/jackrabbit/util/ISO8601.*|org/apache/jackrabbit/util/XMLChar.*",
-                            org.apache.sling.commons.osgi;inline="org/apache/sling/commons/osgi/PropertiesUtil.*",
-                            quartz;inline="org/quartz/CronExpression.*|org/quartz/ValueSet.*",
-                            org.apache.sling.event.api
-                        </Embed-Dependency>
-                    </instructions>
+                    <artifactItems>
+                        <artifactItem>
+                            <groupId>org.apache.sling</groupId>
+                            <artifactId>org.apache.sling.event.api</artifactId>
+                            <destFileName>org.apache.sling.event.api.jar</destFileName>
+                        </artifactItem>
+                        <artifactItem>
+                            <groupId>org.quartz-scheduler</groupId>
+                            <artifactId>quartz</artifactId>
+                            <destFileName>quartz.jar</destFileName>
+                        </artifactItem>
+                        <artifactItem>
+                            <groupId>org.apache.sling</groupId>
+                            <artifactId>org.apache.sling.commons.osgi</artifactId>
+                            <destFileName>org.apache.sling.commons.osgi.jar</destFileName>
+                        </artifactItem>
+                        <artifactItem>
+                            <groupId>org.apache.jackrabbit</groupId>
+                            <artifactId>jackrabbit-jcr-commons</artifactId>
+                            <destFileName>jackrabbit-jcr-commons.jar</destFileName>
+                        </artifactItem>
+                    </artifactItems>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
this plugin is configured in parent, remove here.

Reply:
+1, done